config/config.py

```diff
@@ -0,0 +1,22 @@
+import configparser
+import os
+import shutil
+import json
+
+manifest = json.load(open('config/manifest.json'))
+
+APP_DATA_DIR = '/mrs_app_data'
+CONFIG_DIR = os.path.dirname(os.path.realpath(__file__))  # same dir as this file
+VERSION = manifest['model_version']
+
+if os.path.exists(APP_DATA_DIR):
```

Comment:
What happens here in context of AIDE?

Reply:
It looks for the APP_DATA_DIR inside the container the application runs in. The image does not have this folder so it will always create a new one.